install_template/templates/products/postgres-enterprise-manager/base.njk

```diff
@@ -1,6 +1,9 @@
 {% extends "platformBase/" + platformBaseTemplate + '.njk' %}
 {% set packageName %}edb-pem{% endset %}
+{% block postinstallMenu %}-   [Performing the initial configuration](#initial-configuration){% endblock postinstallMenu %}
```

Comment:
Is there a way to pararametize this content so it doesn't need to be entered for the multiple products that have initial configurations? Same question for the heading (row 5). It may be a question of diminishing returns, but it would be nice not to have to enter the boilerplate each time.